policyengine_us/variables/gov/states/mt/tax/income/subtractions/mt_tuition_subtraction.py

```diff
@@ -0,0 +1,15 @@
+from policyengine_us.model_api import *
+
+
+class mt_tuition_subtraction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Montana tuition subtraction"
+    definition_period = YEAR
+    reference = "https://leg.mt.gov/bills/mca/title_0150/chapter_0300/part_0210/section_0200/0150-0300-0210-0200.html"
+    defined_for = StateCode.MT
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.mt.tax.income.subtractions.tuition
+        contributions = add(tax_unit, period, ["investment_in_529_plan"])
```

Comment:
make a person level `investment_in_529_plan_indv` variable 
add an adds function to the `investment_in_529_plan`

in the mt file - take person level variable and cap for each head and spouse
investment_in_529_plan_indv = ...
head_or_spouse = ...
head_or_spouse_inv. = investment_in_529_plan_indv * head_or_spouse
capped_inv = ....

return tax_unit.sum


change variables names if needed, this is just a sugg

Reply:
please create tests that include multiple people and see whether it passes